### ISSUE TITLE
Suggestion improvement

### ISSUE THREAD

**Issue by usb248** [object505/tipso#9]
Hi,

I am back :) .

On your demo site, i have seen this code (and for personal use, I find that it would be better ... ) 
```javascript
if(jQuery(this).hasClass('clicked')){
    jQuery(this).removeClass('clicked');
	jQuery('.beforeShow').tipso('hide');
  } else {
	jQuery(this).addClass('clicked');
	jQuery('.beforeShow').tipso('show');
  }
```

I think that would be a good idea if you will implement a "toggle" method which hide if the tooltip tipso is currently displayed and conversely. Instead of to use adding/remove CSS class like you do currently...

Why not, moreover, two public method "isHide()" and "isShow()" which return true or false... ?

To sum up, the code will be, simply : 
```javascript
    $('.beforeShow').tipso('toggle');
```
best regards.

**Comment by object505** [comment]
Not implementing this :)

**Comment by usb248** [comment]
I am aware but have you intend to do this ?

**Comment by object505** [comment]
I'm planning to make some adjustments and add more options to the plugin. 
I guess maybe I'll put this too.

**Comment by usb248** [comment]
I took your development version of tipso and I just had to add : 
```javascript
toggle : function() {
 this.mode == 'hide' ? this.show() : this.hide();
}
```

Moreover, I also noticed that the 'speed' option isn't used when we use CSS3 animation.
Speed CSS3 animation always depends of animation-duration in animate.css ...
	
Furthermore, a method to 'refresh' the absolute position of the tooltip will be very useful.
I saw in your code that you only refresh position in show method on windows resize event...

And several other things... i wait your 'some adjustments' :)

**Comment by object505** [comment]
Well feel free to make a PR with you adjustments :)

**Comment by usb248** [comment]
Euhm yes, but i don't know if my adjustments are perfect ... you know better than me your plugin, i'm just here in order to suggest improvement or corrections. 
For example, i don't know if toggle function is fine or not ... it seems to work ^^'
